web/sections/ImageForText/ImageForText.tsx

```diff
@@ -0,0 +1,34 @@
+import { ImageForTextData } from '../../types/types'
+import { forwardRef } from 'react'
+import Image, { Ratios } from '../../pageComponents/shared/SanityImage'
+import Blocks from '../../pageComponents/shared/portableText/Blocks'
+
+type ImageForTextProps = {
+  data: ImageForTextData
+  anchor?: string
+}
+
+const ImageForText = forwardRef<HTMLDivElement, ImageForTextProps>(function ImageForText({ anchor, data }, ref) {
+  const { content, image, aspectRatio } = data
+
+  return (
+    <div ref={ref} id={anchor} className={`${aspectRatio === 'fullWidth' ? 'w-full' : 'px-layout-lg'}`}>
+      <div className="">
+        <Image
+          image={image}
+          maxWidth={2000}
+          {...(aspectRatio === 'fullWidth' && { aspectRatio: Ratios.THREE_TO_TEN })}
```

Comment:
10:3 is not valid for this type

Reply:
Here when using images to 100vw the best ratio was THREE_TO_TEN, so the ratio fullwidth === 3:10